api/docs/http_api_index.md

```diff
@@ -1,13 +1,43 @@
 V1 API Services
 ---------------
 
+The V1 API has been removed. All previous routes can now be found under the
+[Master V3](#Master-V3-HTTP-API) API or the [CA V1](#CA-V1-HTTP-API).
+
+V2 HTTP API
+-----------
+The V2 API has been removed. All previous routes can now be found under the
+[Master V3](#V3-HTTP-API).
```

Comment:
My $.02: I'd totally remove them, because they are removed and not deprecated. That said, it might still make sense on the docs site to retain docs for the 3.7.x-era endpoints. Not sure if that's worth it though, and also not sure what it would take from a markdown/etc perspective.

Related: I'd lean toward not having separate "V1 API Services" and "V2 HTTP API" sections with the repeated verbiage, but just a single "V1/V2 HTTP API" with the verbiage. And that might be where to have a link to the 3.7.x API docs if we go that route.